priv/template/config/config.exs.eex

```diff
@@ -1,4 +1,26 @@
 use Mix.Config
 
-config :exsync,
-  extra_extensions: [".js", ".css"]
+<%= if @ecto do %>
+config :<%= @name %>,
+  ecto_repos: [<%= @module %>.Repo]
+
+config :<%= @name %>, <%= @module %>.Repo,
+  username: "now",
```

Comment:
wonder how ubiquitous this reference is

Reply:
I'm going to remove this and put in something bland and random, it's just what my local db happened to be configured as.